support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -1184,9 +1194,6 @@ function CheckoutComponent({ geoId }: Props) {
 													.then((response) => response.json())
 													.then((json) => {
 														setPayPalBAID((json as { baid: string }).baid);
-														// TODO - this might not meet our browser compatibility requirements (Safari)
```

Comment:
Worth leaving a comment here that says there's a useEffect that submits the form?

Reply:
Though I put it at top on useEffect?

Reply:
Not sure where/what line you would like here, there is a comment on the useEffect itself saying -> `PayPalBAID forces formOnSubmit`

Reply:
Yes, have done